agent/05_agent_configure.sh

```diff
@@ -245,7 +245,104 @@ function generate_cluster_manifests() {
   ansible-playbook -vvv \
           -e install_path=${SCRIPTDIR}/${INSTALL_CONFIG_PATH} \
           "${SCRIPTDIR}/agent/create-manifests-playbook.yaml"
+}
+
+function enable_load_balancer() {
+  if [[ "${PLATFORM_TYPE}" == "none" && "${NUM_MASTERS}" == "3" ]]; then
```

Comment:
Any specific reason to limit just to the compact case?

Reply:
I was just going off of the HA and compact scenarios which are always limited to 3 masters. I'll change it since it is possible for NUM_MASTERS to be any positive number.